scripts/src/java/org/oppia/android/scripts/proto/script_exemptions.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+
+package proto;
+
+option java_package = "org.oppia.android.scripts.proto";
+option java_multiple_files = true;
+
+// Exemptions for the script checks.
```

Comment:
This seems too vague--please add more details.

Reply:
Added.

Reply:
As discussed in meeting, prefer to make these structures & their documentation specific to individual checks since it seems preferable to have distinct textprotos for each exemption list. Keeping the containing proto file generic seems fine.

Ditto for the field name (consider like 'exemptedFilePath' or similar).

Reply:
Done.